velox/functions/sparksql/Arithmetic.h

```diff
@@ -35,15 +35,33 @@ struct RemainderFunction {
     if (UNLIKELY(n == 0)) {
       return false;
     }
-    // std::numeric_limits<int64_t>::min() % -1 could crash the program since
-    // abs(std::numeric_limits<int64_t>::min()) can not be represented in
-    // int64_t.
-    if (UNLIKELY(n == 1 || n == -1)) {
+    if constexpr (std::is_floating_point_v<TInput>) {
+      handle_floating_point(result, a, n);
+    } else {
+      handle_integral(result, a, n);
+    }
+    return true;
+  }
+
+ private:
+  template <typename TInput>
+  inline void handle_floating_point(TInput& result, const TInput& a, const TInput& n) {
```

Comment:
A member function defined inside a class or struct definition is implicitly inline, so I assume we don't need the inline keyword.

Reply:
thanks~,done